packages/create-next-app/create-app.ts

```diff
@@ -45,7 +45,7 @@ export async function createApp({
     try {
       repoUrl = new URL(example)
     } catch (error) {
-      if (error.code !== 'ERR_INVALID_URL') {
+      if (error instanceof Error && error.code !== 'ERR_INVALID_URL') {
```

Comment:
I suggest

 “code” in error && error.code === FOO

Reply:
Using `in` of `unknown` is not valid.

Combined with `error instanceof Error` it leads to `Property 'code' does not exist on type 'never'.`